### DISCUSSION TITLE
Docker compose

### DISCUSSION THREAD

**Discussion by mat926** [simple-login/app#1903] [Self-hosting question]
Is there an official docker compose i can use for installing?

**Comment by springcomp** [comment]
There are a couple of ongoing works but I could not find a definitive status.
Here is [a simple one I’m working on](https://github.com/springcomp/self-hosted-simplelogin/blob/main/simple-login-compose.yaml) and works quite well for my purpose.

**Reply by mat926** [reply]
I have tried your method. Unfortunately I'm running into a problem where I can't create an account or login. Nothing happens when I click the Create Account/Login buttons.  I don't get any errors from the logs either.

**Reply by springcomp** [reply]
Please post an issue and I will assist you.

**Reply by mat926** [reply]
https://github.com/simple-login/app/issues/1910

**Comment by MuhammadNaeem42** [comment]
i will help you to setup this application 
contact me https://www.fiverr.com/s/9W3PNA
[
![simple](https://github.com/simple-login/app/assets/106573678/c5e67f1f-0bc0-49bc-a3a1-406e9e4b0086)
